test/parallel/test-process-execve-no-args.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { skip, isWindows, isIBMi } = require('../common');
+const { fail } = require('assert');
+const { isMainThread } = require('worker_threads');
+const { spawnSync } = require('child_process');
+
+if (!isMainThread) {
+  skip('process.execve is not available in Workers');
```

Comment:
I see this is also used in other test files for `process.execve()` and it was suggested in https://github.com/nodejs/node/pull/56496#discussion_r1905035954, but I don't understand why it is needed? The test does not use worker threads nor `node:test`.

Reply:
We run the entire test suite with `test.py --worker` in [node-test-commit-custom-suites-freestyle (test-worker)](https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/).

Reply:
Oh ok, thanks.